File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        idx=1
        n = len(nums)
        i=0

        for i in range(1,n):
            if (nums[i]!=nums[i-1]):
                nums[idx]=nums[i]
                idx+=1

        return idx
        

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int count = 0;
        int  i = 0;
        int j = 1;

        for (i = 1; i<nums.length; i++){
            if(nums[i-1] != nums[i]){
                nums[j] = nums[i];
                j++;
            }
        }
        return j;
        
    }
}